.github/workflows/ci.yml

```diff
@@ -374,14 +374,13 @@ jobs:
           POSTHOG_API_KEY: ${{ secrets.POSTHOG_API_KEY }}
         run: cd docs/docusaurus && yarn install && bash ../build_docs
 
-      # Temporarily removing as a workaround for https://greatexpectations.atlassian.net/browse/DOC-979
-      # - name: Run broken link checker
-      # uses: lycheeverse/lychee-action@v2
-      # with:
-      # We decided to exclude all external HTTP requests but the ones that under the domain greatexpectations.io
-      # The reason is to avoid having network errors such as pages that throw 429 after too many requests (like Github)
-      # and to prevent other possible errors related to user agent or lychee capturing hrefs from metadata that don't resolve to a specific page (preconnects in JS)
-      #  args: "--exclude='http.*' --include='^https://(.+\\.)?greatexpectations\\.io/' 'docs/docusaurus/build/**/*.html'"
+      - name: Run broken link checker
+        uses: lycheeverse/lychee-action@v2
+        with:
+          # We decided to exclude all external HTTP requests but the ones that under the domain greatexpectations.io
+          # The reason is to avoid having network errors such as pages that throw 429 after too many requests (like Github)
+          # and to prevent other possible errors related to user agent or lychee capturing hrefs from metadata that don't resolve to a specific page (preconnects in JS)
+        args: "--exclude='http.*' --include='^https://(.+\\.)?greatexpectations\\.io/' 'docs/docusaurus/build/**/*.html'"
```

Comment:
`args` needs to be indented 2 spaces so it is inside the `with:` heading. This is what this blocked looked like before it was commented out: https://github.com/great-expectations/great_expectations/blob/35586710279b1dd0c67a61afdfcb50bcb6e444e1/.github/workflows/ci.yml#L383